### PR TITLE
fix: add favicon test asset

### PR DESCRIPTION
**Pull request by mathiasbynens**
The favicon was taken from https://github.com/mathiasbynens/small.

Issue: https://bugzilla.mozilla.org/show_bug.cgi?id=1682076

### CODE REVIEW

**Comment by whimboo** [comment]
Thanks a lot for adding the favicon! Please note that at least the above mentioned test will now pass in Firefox. I would expect that others might also pass. So `itFailsFirefox` would have to be removed too.

File: test/assets/offscreenbuttons.html [hunk 1]
```diff
@@ -29,8 +30,9 @@
 <button id=btn9>9</button>
 <button id=btn10>10</button>
 <script>
-window.addEventListener('DOMContentLoaded', () => {
```
Comment (mathiasbynens) [inline test/assets/offscreenbuttons.html:32]: I wasn't around when this test was created, but I imagine it was written this way assuming the favicon warning would be logged before `DOMContentLoaded`. Now that we have a favicon, we no longer need to try and avoid the error message.

**Comment by mathiasbynens** [comment]
@whimboo Done! Please mark review+ through GitHub's UI once everything looks okay.

**Review by whimboo — APPROVED** [review]
Looks fine from my side, but as I'm not a proper reviewer for this repo you might want to seek someone else too.

**Review by jackfranklin — APPROVED** [review]

**Pull request merged**